content/en/developers/marketplace/marketplace-agent-integration.md

```diff
@@ -0,0 +1,442 @@
+
+`metadata.csv`
+: This contains the list of all metrics collected by your Agent Check. For more information, see the [Metrics metadata file reference][16].
+
+`service_check.json`
+: This contains the list of all Service Checks collected by your Agent Check. Add any service checks you created to the file. For more information, see the [Service check file reference][17].
+
+README
+: Includes an Overview, Setup, Data Collected (optional), and Support section using H2 headings (##) in Markdown.<br><br>For more information, see [README](#readme).
+
+Media
+: Add any images and a video that you want to use for the integration tile's media carousel in an `images` folder. You can add one video to each listing.<br><br>Technology Partners can use `.png` files instead of `.jpg` files to reduce image compression.
+
+Media Carousel
+: Define the images you want to add to the integration tile's media carousel in the `media` object specified in the `tile` front matter on the `manifest.json` file.<br><br>For more information, see [Media Carousel](#media-carousel).
+
+Dashboards and Monitors
+: Provide the JSON files for out-of-the-box dashboards and monitors included in the integration in a `dashboards` folder nested in the `assets` directory.<br><br>Technology Partners can create dashboards and monitors in a provisioned sandbox account, and export these assets into JSON files. For more information about dashboards, see [Best Practices for Integration Preset Dashboards][18].
+
+Logos
+: Add at least one SVG file which can be used in light and dark modes in a `logos` folder nested in the `assets` directory, or add the file(s) directly to the `assets` directory.<br><br>Technology Partners are responsible for the licensing of submitted logos.
+
+CHANGELOG
+: Document release notes and version information in the `CHANGELOG.md` file using the following format: `1.0.0 / YYYY-MM-DD`. This information is displayed in the **Release Notes** tab of the integration tile.<br><br>Technology Partners can add releases and version updates in descending order (latest version at the top).
+
+CODEOWNERS
+: The `CODEOWNERS` file belongs in the shared `.github` directory and defines the individuals or teams responsible for maintaining the content and source code in the Marketplace repository.<br><br>For information on CODEOWNER syntax, see the [GitHub code owners documentation][19].
+
+End User License Agreement (EULA)
+: Add the `eula.pdf` file in the `assets` directory.<br><br>Technology Partners are responsible for adding the EULA.
+
+### README
+
+{{% integrations_readme %}}
+
+### Media carousel
+
+{{% integrations_media_carousel %}}
+
+## Build the wheel
+
+The `pyproject.toml` file provides the metadata that is used to package and build the wheel. The wheel contains the files necessary for the functioning of the integration itself, which includes the Check, configuration example file, and artifacts generated during the build of the wheel.
```

Comment:
"Functioning of the integration" sounded off to me, but feel free to reword or ignore!

```suggestion
The `pyproject.toml` file provides the metadata that is used to package and build the wheel. The wheel contains the files necessary for the integration to function itself, and includes the Check, configuration example file, and artifacts generated during the build of the wheel.
```